### PR TITLE
Upgrade kinto-changes

### PR DESCRIPTION
**Pull request by leplatrem**
I created a `2.X` branch to ease maintenance of stable version (`master`)

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,10 +4,48 @@ CHANGELOG
...
+==================
+
+Configuration Breaking Changes
+''''''''''''''''''''''''''''''
+
+Before:
+
+.. code-block :: ini
+
+    kinto.event_listeners = changes
```
Comment (Natim) [inline CHANGELOG.rst]: I guess we need to apply this changes to the test config file too.

**Comment by leplatrem** [comment]
```
xml-verifier 'https://blocklist.addons.mozilla.org/blocklist/3/\{ec8030f7-c20a-464f-9b0e-13a3a9e97384\}/46.0/' 'http://localhost:8888/v1/blocklist/3/\{ec8030f7-c20a-464f-9b0e-13a3a9e97384\}/46.0/'
/home/travis/gopath/src/github.com/mozilla-services/kinto-dist/.venv/local/lib/python2.7/site-packages/requests/packages/urllib3/util/ssl_.py:334: SNIMissingWarning: An HTTPS request has been made, but the SNI (Subject Name Indication) extension to TLS is not available on this platform. This may cause the server to present an incorrect TLS certificate, which can cause validation failures. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  SNIMissingWarning
  /home/travis/gopath/src/github.com/mozilla-services/kinto-dist/.venv/local/lib/python2.7/site-packages/requests/packages/urllib3/util/ssl_.py:132: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecurePlatformWarning
  /home/travis/gopath/src/github.com/mozilla-services/kinto-dist/.venv/local/lib/python2.7/site-packages/requests/packages/urllib3/util/ssl_.py:132: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecurePlatformWarning
  "GET   /v1/blocklist/3/%5C%7Bec8030f7-c20a-464f-9b0e-13a3a9e97384%5C%7D/46.0/" 200 (68 ms) request.summary agent=python-requests/2.12.5 authn_type=None errno=None lang=None time=2017-02-15T09:12:15 uid=None
--- /tmp/tmp5CTXbD  2017-02-15 09:12:15.717567856 +0000
+++ /tmp/tmpJ8GI7r  2017-02-15 09:12:15.717567856 +0000
@@ -7355,36 +7355,25 @@
                     }, 
                     "versionRange": {
                         "@maxVersion": "18.0.0.366", 
                         "@minVersion": "18.0.0.360", 
                         "@severity": "0", 
                         "@vulnerabilitystatus": "1"
                     }
                 }, 
                 {
                     "@blockID": "p1274", 
                     -                    "@os": "", 
                     "infoURL": "https://get.adobe.com/flashplayer/", 
                     -                    "match": [
-                        {
-                            "@exp": "", 
-                            "@name": "description"
-                        }, 
-                        {
-                            "@exp": "(NPSWF32.*\\.dll)|(NPSWF64.*\\.dll)|(Flash\\ Player\\.plugin)", 
-                            "@name": "filename"
-                        }, 
-                        {
-                            "@exp": "", 
-                            "@name": "name"
-                        }
-                    ], 
+                    "match": {
+                        "@exp": "(NPSWF32.*\\.dll)|(NPSWF64.*\\.dll)|(Flash\\ Player\\.plugin)", 
+                        "@name": "filename"
+                    }, 
                     "versionRange": {
                         "@maxVersion": "22.0.0.211", 
                         "@minVersion": "22.0.0.192", 
                         "@severity": "0", 
                         "@vulnerabilitystatus": "1"
                     }
                 }, 
                 {
                     "@blockID": "p129", 
                     "match": {
                     @@ -7437,159 +7426,96 @@
                         "@maxVersion": "18.0.0.382", 
                         "@minVersion": "18.0.0.366", 
                         "@severity": "0", 
                         "@vulnerabilitystatus": "1"
                     }
                 }, 
                 {
                     "@blockID": "p1419", 
                     "@os": "Linux", 
                     "infoURL": "https://get.adobe.com/flashplayer/", 
                     -                    "match": [
-                        {
-                            "@exp": "", 
-                            "@name": "description"
-                        }, 
-                        {
-                            "@exp": "libflashplayer\\.so", 
-                            "@name": "filename"
-                        }, 
-                        {
-                            "@exp": "", 
-                            "@name": "name"
-                        }
-                    ], 
+                    "match": {
+                        "@exp": "libflashplayer\\.so", 
+                        "@name": "filename"
+                    }, 
                     "versionRange": {
                         "@maxVersion": "11.2.202.643", 
                         "@minVersion": "11.2.202.637", 
                         "@severity": "0", 
                         "@vulnerabilitystatus": "1"
                     }
                 }, 
                 {
                     "@blockID": "p1420", 
                     -                    "@os": "", 
                     "infoURL": "https://get.adobe.com/flashplayer/", 
                     -                    "match": [
-                        {
-                            "@exp": "", 
-                            "@name": "description"
-                        }, 
-                        {
-                            "@exp": "(NPSWF32.*\\.dll)|(NPSWF64.*\\.dll)|(Flash\\ Player\\.plugin)", 
-                            "@name": "filename"
-                        }, 
-                        {
-                            "@exp": "", 
-                            "@name": "name"
-                        }
-                    ], 
+                    "match": {
+                        "@exp": "(NPSWF32.*\\.dll)|(NPSWF64.*\\.dll)|(Flash\\ Player\\.plugin)", 
+                        "@name": "filename"
+                    }, 
                     "versionRange": {
                         "@maxVersion": "23.0.0.205", 
                         "@minVersion": "23.0.0.185", 
                         "@severity": "0", 
                         "@vulnerabilitystatus": "1"
                     }
                 }, 
                 {
                     "@blockID": "p1421", 
                     "@os": "Linux", 
                     "infoURL": "https://get.adobe.com/flashplayer/", 
                     -                    "match": [
-                        {
-                            "@exp": "", 
-                            "@name": "description"
-                        }, 
-                        {
-                            "@exp": "libflashplayer\\.so", 
-                            "@name": "filename"
-                        }, 
-                        {
-                            "@exp": "", 
-                            "@name": "name"
-                        }
-                    ], 
+                    "match": {
+                        "@exp": "libflashplayer\\.so", 
+                        "@name": "filename"
+                    }, 
                     "versionRange": {
                         "@maxVersion": "23.0.0.207", 
                         "@minVersion": "11.2.202.643", 
                         "@severity": "0", 
                         "@vulnerabilitystatus": "1"
                     }
                 }, 
                 {
                     "@blockID": "p1422", 
                     -                    "@os": "", 
                     "infoURL": "https://get.adobe.com/flashplayer/", 
                     -                    "match": [
-                        {
-                            "@exp": "", 
-                            "@name": "description"
-                        }, 
-                        {
-                            "@exp": "(NPSWF32.*\\.dll)|(NPSWF64.*\\.dll)|(Flash\\ Player\\.plugin)", 
-                            "@name": "filename"
-                        }, 
-                        {
-                            "@exp": "", 
-                            "@name": "name"
-                        }
-                    ], 
+                    "match": {
+                        "@exp": "(NPSWF32.*\\.dll)|(NPSWF64.*\\.dll)|(Flash\\ Player\\.plugin)", 
+                        "@name": "filename"
+                    }, 
                     "versionRange": {
                         "@maxVersion": "23.0.0.207", 
                         "@minVersion": "23.0.0.205", 
                         "@severity": "0", 
                         "@vulnerabilitystatus": "1"
                     }
                 }, 
                 {
                     "@blockID": "p1494", 
                     -                    "@os": "", 
                     "infoURL": "https://get.adobe.com/flashplayer/", 
                     -                    "match": [
-                        {
-                            "@exp": "", 
-                            "@name": "description"
-                        }, 
-                        {
-                            "@exp": "(NPSWF32.*\\.dll)|(NPSWF64.*\\.dll)|(Flash\\ Player\\.plugin)", 
-                            "@name": "filename"
-                        }, 
-                        {
-                            "@exp": "", 
-                            "@name": "name"
-                        }
-                    ], 
+                    "match": {
+                        "@exp": "(NPSWF32.*\\.dll)|(NPSWF64.*\\.dll)|(Flash\\ Player\\.plugin)", 
+                        "@name": "filename"
+                    }, 
                     "versionRange": {
                         "@maxVersion": "24.0.0.186", 
                         "@minVersion": "23.0.0.207", 
                         "@severity": "0", 
                         "@vulnerabilitystatus": "1"
                     }
                 }, 
                 {
                     "@blockID": "p1495", 
                     "@os": "Linux", 
                     "infoURL": "https://get.adobe.com/flashplayer/", 
                     -                    "match": [
-                        {
-                            "@exp": "", 
-                            "@name": "description"
-                        }, 
-                        {
-                            "@exp": "libflashplayer\\.so", 
-                            "@name": "filename"
-                        }, 
-                        {
-                            "@exp": "", 
-                            "@name": "name"
-                        }
-                    ], 
+                    "match": {
+                        "@exp": "libflashplayer\\.so", 
+                        "@name": "filename"
+                    }, 
                     "versionRange": {
                         "@maxVersion": "24.0.0.186", 
                         "@minVersion": "23.0.0.207", 
                         "@severity": "0", 
                         "@vulnerabilitystatus": "1"
                     }
                 }, 
                 {
                     "@blockID": "p152", 
                     "match": {
```

**Comment by Natim** [comment]
Ping @jvillalobos this refs: https://bugzilla.mozilla.org/show_bug.cgi?id=1339500

I really think we should remove the empty fields using the kinto-admin.

@n1k0 has been working on a patch for that: https://github.com/mozilla-services/react-jsonschema-form/pull/472

**Comment by Natim** [comment]
Ok the bug have been fixed. I guess we should start removing the xml-verifier here because we are no longer using json2kinto to sync between AMO and Kinto.

File: setup.py [hunk 2]
```diff
@@ -31,7 +31,7 @@ def read_file(filename):
 DEPENDENCY_LINKS = []
 
 setup(name='kinto-dist',
-      version='1.13.0.dev0',
+      version='2.0.0.dev0',
```
Comment (Natim) [inline setup.py]: If we are getting a 2.0 there, I would like to merge the remove-py27 support and upgrade to Python > 3.5 and PostgreSQL > 9.5 at the same time.

**Pull request merged**